README.md

```diff
@@ -1,159 +1,163 @@
-
-- Mounting a QoS enabled volume can take longer than a volume without QoS for both FC and iSCSI protocol.
-
-- For a cloned volume with the same size as source volume, comment field won’t be populated on 3PAR.
-
-- User not allowed to import a 3PAR legacy volume when it is under use(Active VLUN).
-
-- User needs to explicitly manage all the child snapshots, until which managed parent volume cannot be deleted
-
-- User cannot manage already managed volume by other docker host(i.e. volume thats start with 'dcv-')
-
-- It is recommended for a user to avoid importing legacy volume which has schedules associated with it. If this volume needs to be imported please remove existing schedule on 3PAR and import the legacy volume.
-
-- "Snapshot schedule creation can take more time resulting into Docker CLI timeout. However, snapshot schedule may still get created in the background. User can follow below two steps in case of time out issue from docker daemon while creating snapshot schedule."
+* These are Ansible playbooks to automate the install of the HPE 3PAR Docker Volume Plug-in for Docker for use within standalone docker environment or Kubernetes/OpenShift environments.
+```
+NOTE: The Ansible installer only supports Ubuntu/RHEL/CentOS. 
+If you are using another distribution of Linux, you will need to modify the 
+playbooks to support your application manager (apt, etc.) and the pre-requisite packages.
+```
+These playbooks perform the following tasks on the Master/Worker nodes as defined in the Ansible [hosts](https://github.com/hpe-storage/python-hpedockerplugin/blob/master/ansible_3par_docker_plugin/hosts) file.
+* Configure the Docker Services for the HPE 3PAR Docker Volume Plug-in.
+* Deploys the config files (iSCSI or FC) to support your environment.
+* Installs the HPE 3PAR Docker Volume Plug-in (Containerized version).
+* For standalone docker environment, deploys an HPE customized etcd cluster.
+* For Kubernetes/OpenShift, deploys a Highly Available HPE etcd cluster used by the HPE 3PAR Docker Volume plugin.
+* Supports single node (Use only for testing purposes) or multi-node deployment (HA) as defined in the Ansible hosts file.
+* Deploys the HPE FlexVolume Driver.
+* FlexVolume driver deployment for single master and multimaster will be as per the below table.
+
+Cluster       | OS 3.9        | OS 3.10        | OS 3.11    | K8S 1.11      |  K8S 1.12     | K8S 1.13     | K8S 1.14     | K8S 1.15
+------------- | ------------- | -------------  | -----------|------------   |-------------  |------------- |------------- | -------------
+Single Master | System Process| System Process | Deployment | System Process| System Process| Deployment   | Deployment   | Deployment
+Multimaster   | NA            | NA             |  Deployment| NA            | NA            | Deployment   | Deployment  | Deployment 
+                
+```
```

Comment:
Add the note for the system process and deployment 
Move this existing note for the docker version in the respective supported docker version section(system requirement)